boards/nullbitsco/tidbit/README.md

```diff
@@ -27,7 +27,7 @@ from kmk.extensions.rgb import RGB, AnimationModes
 keyboard = KMKKeyboard(active_encoders=[0], landscape_layout=True)
```

Comment:
This is not correct. There is no `landscape_layout` parameter.

Reply:
There is for the tidbit, but not for the nibble.  See here: https://github.com/KMKfw/kmk_firmware/blob/main/boards/nullbitsco/tidbit/kb.py#L48